bin/mocha

```diff
@@ -16,7 +16,7 @@ var spawn = require('child_process').spawn,
 getOptions();
 
 process.argv.slice(2).forEach(function(arg){
-  var flag = arg.split('=')[0];
+  var flag = arg.split('=', 1)[0].replace(/([a-z])_/g, '$1-');
```

Comment:
would prefer no lines are *changed* in this file.  it appears you can add two new cases for `--es-staging` and `--use-strict` to the hideous `switch` below; this seems like a safer bet.